docs/en/api/REST-API.md

```diff
@@ -21,8 +21,8 @@ can use in-memory streams, the REST API decouples the stream creation and access
 `create` and `open` REST API methods and the `streams` resource endpoints for details).
 
 The HTTP proxy is a standalone server that can be started using
-`${ALLUXIO_HOME}/bin/alluxio-start.sh proxy` and stopped using `${ALLUXIO_HOME}/bin/alluxio-stop.sh
-proxy`. By default, the REST API is available on port 39999.
+`${ALLUXIO_HOME}/bin/alluxio process start proxy` and stopped using `${ALLUXIO_HOME}/bin/alluxio process stop proxy`.
```

Comment:
talked to @JiamingMai and i think the proxy is no longer support in 30x.

Reply:
we still have the code for the process; is there a plan to remove it?
should this doc be completely removed/replaced? @LuQQiu i remember you mentioned the rest api was being replaced; any suggestions on how to handle the doc page?